scripts/google_replicate.py

```diff
@@ -21,12 +21,12 @@
 
 from scripts import utils
 from scripts.errors import APIError, UserError, StreamError
-from scripts.settings import PROJECT_ACL, INDEXD, GDC_TOKEN, IGNORED_FILES
+from scripts.settings import PROJECT_ACL, INDEXD, GDC_TOKEN, IGNORED_FILES, DATA_ENDPT
 from scripts import indexd_utils
 
 logger.basicConfig(level=logger.INFO, format="%(asctime)s %(message)s")
 
-DATA_ENDPT = "https://api.gdc.cancer.gov/data/"
+DATA_ENDPT = DATA_ENDPT
```

Comment:
Cool 👍 
Maybe we don't even need this line, right?

Reply:
You are correct! Haha, didnt even think about it. I could just get rid of it.